src/hooks/useRadio.ts

```diff
@@ -20,15 +21,22 @@ export const useRadio = (audio?: HTMLAudioElement) => {
   }, [volume]);
 
   useEffect(() => {
-    if (audio) {
+    if (audio && playRadio) {
       audio.volume = volume / 100;
       audio?.play();
+
+      return () => {
+        audio.volume = 0;
+        audio.remove();
+      };
     }
-  }, [volume, audio]);
+  }, [volume, audio, playRadio]);
 
   useEffect(() => {
-    audio?.play();
-  }, [audio]);
+    if (playRadio) {
+      audio?.play();
+    }
+  }, [audio, playRadio]);
```

Comment:
This second `useEffect` seems to be using only a part of the logic that's in the one above. The dependency arrays are the same. Is this even required?

Reply:
Looks like it's redundant from a quick glance? Let's get this cleared up I reckon.